### ISSUE TITLE
OrderSummary (React) breaks when updating to v9 of cc

### ISSUE THREAD

**Issue by cknabe** [carbon-design-system/carbon#887]
I'm currently updating the Gen1 order form package.json to have newer versions of different modules.
https://github.ibm.com/Bluemix/Gen1.VSI

The problem I'm seeing could potentially even be outside of carbon-components. It is possible I need to clear some cache, or possible that when I updated webpack it is missing something. 

We've been using ```    "carbon-components": "8.16.8",
    "carbon-components-react": "5.31.0",``` 
I get the screenshot down below when I update to: ```    "carbon-addons-cloud": "^2.0.1",
    "carbon-addons-cloud-react": "^1.1.4",
    "carbon-components": "^9.0.2",
    "carbon-components-react": "^6.4.2",```

Original: 
![image](https://user-images.githubusercontent.com/13582620/41254279-b1b9931c-6d88-11e8-8707-3025850ac114.png)

After updating to v9 in package.json:
![image](https://user-images.githubusercontent.com/13582620/41254308-c10e684c-6d88-11e8-961d-364754022992.png)

When I bump carbon-components v9 back to the v8 version, everything is good again. I am a little unsure of how carbon-components-react is resolving dependencies when it requires a peer dependency of v9 for carbon-components, but I have specified v8.

**Comment by cknabe** [comment]
With v9 installed it shows our app as using it: 
```
$ npm ls carbon-components
gen1-vsi@0.0.0-semantic-release C:\Users\IBM_ADMIN\git\Gen1.VSI
`-- carbon-components@9.0.2
```

With v8 installed it doesn't show up:
```
$ npm ls carbon-components
gen1-vsi@0.0.0-semantic-release C:\Users\IBM_ADMIN\git\Gen1.VSI
`-- (empty)
```

**Comment by asudoh** [comment]
Hi @cknabe thank you for reporting! I tried to upgrade the dev env of `carbon-addons-cloud-react` to V9 of `carbon-components` and the corresponding `carbon-components-react` version, but couldn’t reproduce the problem. The dev env of `carbon-addons-cloud-vanilla` has been upgraded to V9 of `carbon-components` already, but couldn’t reproduce the problem there, either. Could you try re-creating the problem in CodePen or CodeSandbox? Thanks!

**Comment by tw15egan** [comment]
Closing due to inactivity, but feel free to reopen if you feel this is still an issue

**Comment by cknabe** [comment]
This can be closed with 'non-issue'

Sorry, I apologize for missing the original reply. I bit the bullet though and completely upgraded our app to carbon-components v9.
In the process I discovered a lot of css overrides that were previously written to hack some of the carbon components stuff into place. I can't remember the specifics, but this OrderSummary issue was due to a difference in flex.
Thank you for trying @asudoh
